pkg/deploy/api/v1/types.go

```diff
@@ -115,9 +116,32 @@ type RollingDeploymentStrategyParams struct {
 	// TimeoutSeconds is the time to wait for updates before giving up. If the
 	// value is nil, a default will be used.
 	TimeoutSeconds *int64 `json:"timeoutSeconds,omitempty" description:"the time to wait for updates before giving up"`
+	// The maximum number of pods that can be unavailable during the update.
```

Comment:
I know these are already in like this upstream but lets keep golint happy here: `// MaxUnavailable is the ..`